swagger/specification/applications/resource-manager/Applications.Core/preview/2023-10-01-preview/openapi.json

```diff
@@ -3639,6 +3639,26 @@
         }
       }
     },
+    "EnvironmentVariables": {
```

Comment:
I think we need to explicitly document the format env variable key is expected in for secret values, since it would not be obvious to someone new https://github.com/radius-project/design-notes/blob/4a768094e5541e3b1ab1716c848a06f5eb21d09f/recipe/2024-02-terraform-providers.md?plain=1#L186

Reply:
It is documented both in design doc [link](https://github.com/radius-project/design-notes/blob/4a768094e5541e3b1ab1716c848a06f5eb21d09f/recipe/2024-02-terraform-providers.md?plain=1#L343) and in this file as well (ln3650, ln4463). Let me know if I didn't read the question right.

Reply:
It is understandable from the example in the design doc, but I'm not sure where is it explicitly described in the typespec, could you point me add a link to the line? I think we are likely talking about different things, L3650 is ProviderSecret, I'm looking for this the description for where env variable key needs to be defined - for example `MY_ENV_VAR_2` from the design doc here: https://github.com/radius-project/design-notes/blob/4a768094e5541e3b1ab1716c848a06f5eb21d09f/recipe/2024-02-terraform-providers.md?plain=1#L259

Reply:
[Ln124](https://github.com/vishwahiremat/radius/pull/1/files#diff-afd83699015b6426efe6ec70fcec90303e63689cef1b48839eb95d8435acd046R124) will denote map of string and the Secret type

Reply:
> [Ln124](https://github.com/vishwahiremat/radius/pull/1/files#diff-afd83699015b6426efe6ec70fcec90303e63689cef1b48839eb95d8435acd046R124) will denote map of string and the Secret type

How does a user who is authoring bicep figures out what the "string" in this map is meant for?